src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/NullToVisibilityConverter.cs

```diff
@@ -26,5 +33,10 @@ public object ConvertBack(object value, Type targetType, object parameter, Cultu
             Debug.Fail("Not Implemented");
             return null;
         }
+
+        /// <summary>
+        /// If a string is null or empty, collapse it. By default, it will only collapse null values (including for strings)
+        /// </summary>
+        public bool CollapseEmptyString { get; set; }
```

Comment:
to be accessed by xaml, members generally need to be public. Converters are generally built to be used. 
I'm refreshing my memory to see if it is technically possible...it might be: https://docs.microsoft.com/en-us/dotnet/desktop-wpf/xaml-services/define-custom-types#access-level

Reply:
i tested just switching it...and it does work...(me in a former life helped make that possible).
But since you dropped by and said, not important, i reverted it.